attributes/_user_management.rb

```diff
@@ -4,6 +4,7 @@
 if node['platform_family'] == 'windows'
   default['splunk']['user'] = 'SYSTEM'
   default['splunk']['group'] = 'SYSTEM'
+  default['splunk']['password'] = 'splunk'
```

Comment:
Guaranteed this isn't the SYSTEM password... so it shouldn't be set this way,

But also should we run Splunk as a different user on Windows?

Reply:
I have changed it to be something a consumer can store in an encrypted data bag.